scripts/cli-get-changed-files.js

```diff
@@ -0,0 +1,96 @@
+const config = require('../config');
+const getChangedFilesBitbucketPipelines = require('./cli-get-changed-files-bitbucket-pipelines');
```

Comment:
@allmywant @kingo55 not sure if it's a good idea, but could the value of `getChangedFilesBitbucketPipelines` be tied to the `supportedCIs` array, instead of being it's own constant?
We could then change the logic on line 65 to set `getChangedFilesFn` to suit.

Reply:
Do you mean like how it's done in `scripts/cli-send-lifecycle-events.js`?

Reply:
not quite, I was thinking of something like:
```
const supportedCIs = {
    'bitbucket-pipelines': {
        changedFilesFn: './cli-get-changed-files-bitbucket-pipelines'
    }
};

...

getChangedFilesFn = require(supportedCIs[ciEnv].changedFilesFn);
```

Reply:
@dapperdrop @kingo55 
I did it like that because I wanted to do configuration check in scripts/cli-get-changed-files.js, since it's the first function in the call stack so it's the best time to check the configuration.
I will change the code like this when we start implement more CI systems:
supportedCIs = {
   'bitbucket-pipelines': require('./cli-get-changed-files-bitbucket-pipelines'),
   ...
}

Reply:
@allmywant cool - noted. Happy to go with your approach

Reply:
I'll mark this as resolved for now.